### DISCUSSION TITLE
UsePrism .NET 8.0 [9.0.401-pre]

### DISCUSSION THREAD

**Discussion by MarcinGaspol** [PrismLibrary/Prism#3116] [.NET MAUI]
MauiApp CreateMauiApp for Android what add to UsePrism  for param IContainerExtension ?

**Comment by giuseppenovielli** [comment]
See these follow links 

https://github.com/giuseppenovielli/Prism-MAUI-Samples
https://docs.prismlibrary.com/docs/platforms/maui/appbuilder.html

**Reply by dansiegel** [reply]
@MarcinGaspol if you aren't seeing a version without the IContainerExtension then you need to add a reference to Prism.DryIoc.Maui. Only those with a Commercial Plus license have access to the additional containers.
